### DISCUSSION TITLE
Im  a little bit confused from the docs installation manual

### DISCUSSION THREAD

**Discussion by oxidian24** [explosion/spaCy#12225] [Help: Installation]
Hello, I installed spacy but I am a little bit confused from the doc. Can you check it if everything is all right? So I installed the pipelines, but is the file structure in .env OK?

I dont know what to do with the downloaded files in .env folder. Should they be copied to .env folder?

(https://youtu.be/SBEXpqUTZvM

**Comment by adrianeboyd** [comment]
This looks like a mistake, should we delete this discussion?

**Reply by oxidian24** [reply]
No responses came so you can delete it. But i still dont know what to do with the downloaded files in .env folder. Should they be copied to .env folder?

**Reply by rmitsch** [reply]
Sorry, we aren't able to respond to questions in this format or provide support at the level of verifying individual installations. If you have a concrete question / installation issue, please include the full question in your post, using markdown blocks for code and output/tracebacks (no videos).
